ui/app/src/Router.tsx

```diff
@@ -18,7 +18,7 @@ import Docs from './views/Docs';
 function Router() {
   return (
     <Routes>
-      <Route path="/" element={<App />} />
+      <Route path="/projects/:projectID/dashboards/:dashboardID" element={<App />} />
```

Comment:
probably `App` should be reviewed, so the loading of the header, footer are done at a upper level

Reply:
A redirect or temporary landing page would be nice at some point, I saw you mentioned it in the PR description but I was still a little confused when I landed on this by default:
![image](https://user-images.githubusercontent.com/9369625/196784716-d42de31c-3ef2-42fd-8104-26ad6504b3ac.png)

Reply:
I'm thinking of having the list of dashboard at "/", like that we would be able to navigate between each dashboard without entering manually the URL

Reply:
I reorganised the App in order to make this file the real entry point of the application.

Like that every page will contain the header and the footer.

Reply:
Temporary I make the root page pointed to the `ViewDashboard` but I would make this change in a later PR.